featuretools/tests/testing_utils/mock_ds.py

```diff
@@ -115,7 +115,7 @@ def make_ecommerce_dataframes(with_integer_time_index=False):
         'engagement_level': [1, 3, 2],
         'full_name': ['Mr. John Doe', 'Doe, Mrs. Jane', 'James Brown'],
         'email': ['john.smith@example.com', np.nan, 'team@featuretools.com'],
-        'phone_number': ['5555555555', '555-555-5555', '1-(555)-555-5555'],
+        'phone_number': ['555-555-5555', '555-555-5555', '1-(555)-555-5555'],
```

Comment:
Why does this need to be changed?

Reply:
This is because of the `OverflowError`. The string `'5555555555'` is now getting interpreted as numeric when it gets compared to datetimes with something like `df['phone_numbers'] == df['dates']` in Dask